codegen/projections/white_label/lib/white_label/config.rb

```diff
@@ -174,7 +174,7 @@ def defaults
         http_api_key_provider: [proc { |cfg| cfg[:stub_responses] ? Hearth::IdentityProvider.new(proc { Hearth::Identities::HTTPApiKey.new(key: 'stubbed api key') }) : nil }],
         http_bearer_provider: [proc { |cfg| cfg[:stub_responses] ? Hearth::IdentityProvider.new(proc { Hearth::Identities::HTTPBearer.new(token: 'stubbed bearer') }) : nil }],
         http_client: [proc { |cfg| Hearth::HTTP::Client.new(logger: cfg[:logger]) }],
-        http_custom_auth_provider: [proc { |cfg| cfg[:stub_responses] ? Hearth::IdentityProvider.new(proc { Auth::HTTPCustomAuthIdentity.new(key: 'key') }) : nil }],
+        http_custom_auth_identity_provider: [proc { |cfg| cfg[:stub_responses] ? Hearth::IdentityProvider.new(proc { Auth::HTTPCustomAuthIdentity.new(key: 'key') }) : nil }],
```

Comment:
Revert?

Reply:
The name of the CustomAuthIdentity has identity in it and I think it should, so I included it here since it would otherwise be "auth_provider" which sounds like its providing auth.